### ISSUE TITLE
Context menu actions not working

### ISSUE THREAD

**Issue by smashwilson** [atom/atom#22018]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

I've just noticed that, in the latest nightly build, context menu actions are not working throughout the app. The context menu appears and allows me to choose an option, but nothing happens.

### Steps to Reproduce

1. Open Atom with a project
2. Right-click on a folder in the tree view
3. Choose "New file..."

**Expected behavior:**

The file creation dialog panel should appear.

**Actual behavior:**

The context menu closes and no dialog is shown.

**Reproduces how often:**

100% of the time. It also reproduces with any other menu option on any other context menu (within a TextEditor, or on unstaged files in the git tab, etc).

### Versions

```
github ➤ atom --version
Atom    : 1.57.0-nightly0
Electron: 9.4.4
Chrome  : 83.0.4103.122
Node    : 12.14.1
github ➤ apm --version
apm  2.5.2
npm  6.14.8
node 12.4.0 x64
atom 1.57.0-nightly0
python 2.7.16
git 2.27.0.3050.gc219791418
```

### Additional Information

No errors appear in the dev console, either.

Keyboard shortcuts (like "A" for the "New file" action in my repro steps) and running commands from the command palette both work as expected.

I'm guessing that this is related to an Electron upgrade?

**Comment by darangi** [comment]
Reproduced on nightly. I believe it is related to the electron upgrade

**Comment by aminya** [comment]
> Reproduced on nightly. I believe it is related to the electron upgrade

Could you delay the schedule for the next nightly release? There is also an issue with the CLI:
https://github.com/atom/atom/issues/22024

, and this PR is not merged yet:
https://github.com/atom/atom/pull/21927

**Comment by sadick254** [comment]
@aminya we can't delay the nightly releases because of the bugs mentioned. Nightly builds are a reflection of what is on the `master`. The best we can do is expedite the fixes of the bugs mentioned.

**Comment by UziTech** [comment]
@sadick254 Can we at least get those fixes backported to v1.56.0 before it is released?

**Comment by aminya** [comment]
We should definitely delay the stable version release. It doesn't make sense to release a stable version with a lot of bugs that are already fixed. 

The Electron update should have all the fixes included otherwise people will open a lot of issues here.

**Comment by p-d-boiko** [comment]
> people will open a lot of issues here.

Ruined UX and usability is the issue to worry about, not the github issues. Because it reduces productivity, and thus speeding up users drain to a competition products.

**Comment by asturur** [comment]
So i came here because i m exeperiencing all kind of issues. this is the biggest one to be honest.
I'll see if i find any of the root causes

**Comment by aceventura82** [comment]
I'm having the same problem
Atom Beta 1.56.0-beta0 x64
Manjaro Linux 20.2.1

When running atom from terminal I get this error message each time a click a item in the menu 

`Failed to serialize arguments
Error: Failed to serialize arguments
    at WebContents._.send (electron/js2c/browser_init.js:173:2149)
    at AtomWindow.sendCommandToBrowserWindow (/usr/lib/atom/resources/app.asar/src/main-process/atom-window.js:377:36)
    at AtomWindow.sendCommand (/usr/lib/atom/resources/app.asar/src/main-process/atom-window.js:364:12)
    at AtomApplication.sendCommandToWindow (/usr/lib/atom/resources/app.asar/src/main-process/atom-application.js:1107:27)
    at item.click (/usr/lib/atom/resources/app.asar/src/main-process/context-menu.js:21:34)
    at MenuItem.click (electron/js2c/browser_init.js:65:1624)
    at Function.executeCommand (electron/js2c/browser_init.js:73:1102)
`

Havbe tried fresh install and safe mode but the problem persist

**Comment by asturur** [comment]
So i have a solution that works for me, i m just not sure is the correct one. Thanks @aceventura82 because your stack here reduce the time for me to try to fix it to 5 minutes, rather thank looking everything from scratch.
I'll open the pr with the intent of spinning up a discussion to fix it, not sure my fix is acceptable

**Comment by asturur** [comment]
I opened an alternative PR i did not notice there was another one already open. The fix is indeed different, while the first PR seems a safer approach, i wonder what are we doing with that command detail at all.
Maintainers should know probably, i m interested to understand.

**Comment by darangi** [comment]
Fixed by https://github.com/atom/atom/pull/22046

**Comment by UziTech** [comment]
@darangi can you make sure this gets fixed in v1.56.0 before it goes to stable as well. It would also be nice to get a v1.56.0-beta1 release with this fix.
